onnxruntime/contrib_ops/cuda/math/binary_elementwise_ops_impl.cu

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <cuda_runtime.h>
+#include "binary_elementwise_ops_impl.h"
+#include "core/providers/cuda/cu_inc/common.cuh"
+#include "core/providers/cuda/cu_inc/binary_elementwise_impl.cuh"
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+
+#define OP(name, expr)                                   \
+  template <class T>                                     \
+  struct OP_##name {                                     \
+    __device__ __inline__ T operator()(T a, T b) const { \
+      return (expr);                                     \
+    }                                                    \
+  };
+
+#define CONTRIB_BINARY_ELEMENTWISE_IMPL(name)              \
+  CONTRIB_BINARY_ELEMENTWISE_IMPL_DECLARATION(name) {      \
+    BinaryElementWiseImpl(output_rank_or_simple_broadcast, \
+                          lhs_padded_strides,              \
+                          lhs_data,                        \
+                          rhs_padded_strides,              \
+                          rhs_data,                        \
+                          fdm_output_strides,              \
+                          fdm_H,                           \
+                          fdm_C,                           \
+                          output_data,                     \
+                          OP_##name<T>(),                  \
+                          count);                          \
+  }
+
+#define CONTRIB_SPECIALIZED_BINARY_ELEMENTWISE_IMPL(x, T)                             \
+  template void Impl_##x<T>(size_t output_rank,                                       \
+                            const int64_t* lhs_padded_strides,                        \
+                            const T* lhs_data,                                        \
+                            const int64_t* rhs_padded_strides,                        \
+                            const T* rhs_data,                                        \
+                            const onnxruntime::cuda::fast_divmod* fdm_output_strides, \
+                            const onnxruntime::cuda::fast_divmod& fdm_H,              \
+                            const onnxruntime::cuda::fast_divmod& fdm_C,              \
+                            T* output_data, size_t count);
+
+#define CONTRIB_SPECIALIZED_BINARY_ELEMENTWISE_IMPL_UZILHFD(x) \
```

Comment:
Where did you use these two macros: CONTRIB_SPECIALIZED_BINARY_ELEMENTWISE_IMPL_UZILHFD and CONTRIB_SPECIALIZED_BINARY_ELEMENTWISE_IMPL_OIL

Reply:
they are left for future use. In the future, we can transform all the subgraph with pattern binaryOp + Unary Op

Reply:
Since the macros are not used, it is better to comment them or remove them. You can even create a branch to keep these macro until they are used.